source/includes/client_sdk.md

```diff
@@ -10,7 +10,13 @@
 </head>
 <body>
   <highline-link></highline-link>
-  <script src="https://link.highline.co/v1/highline-link.js"></script>
+
+  <!-- Sandbox -->
+  <script src="https://stg-link.highline.co/highline-link.js"></script>
+
+  <!-- Production -->
+  <!-- <script src="https://link.highline.co/v1/highline-link.js"></script> -->
```

Comment:
Why comment it?

Reply:
Just for clarity that they should have one or the other, not both.

Reply:
<!-- this would be hidden correct?-->